jmqtui/jmqtui/open_wallet_dialog.py

```diff
@@ -92,6 +92,30 @@ def setupUi(self, OpenWalletDialog):
 
         self.verticalLayout.addLayout(self.horizontalLayout_2)
 
+        self.horizontalLayout_4 = QHBoxLayout()
+        self.horizontalLayout_4.setSpacing(10)
+        self.horizontalLayout_4.setObjectName(u"horizontalLayout_4")
+        self.errorMessageLabel = QLabel(OpenWalletDialog)
+        self.errorMessageLabel.setObjectName(u"errorMessageLabel")
+        palette = QPalette()
+        brush = QBrush(QColor(239, 41, 41, 255))
+        brush.setStyle(Qt.SolidPattern)
+        palette.setBrush(QPalette.Active, QPalette.WindowText, brush)
+        palette.setBrush(QPalette.Inactive, QPalette.WindowText, brush)
+        brush1 = QBrush(QColor(190, 190, 190, 255))
+        brush1.setStyle(Qt.SolidPattern)
+        palette.setBrush(QPalette.Disabled, QPalette.WindowText, brush1)
+        self.errorMessageLabel.setPalette(palette)
+        font = QFont()
+        font.setBold(True)
+        font.setWeight(75)
+        self.errorMessageLabel.setFont(font)
+
+        self.horizontalLayout_4.addWidget(self.errorMessageLabel)
+
+
```

Comment:
nit - unnecessary extra blank line

Reply:
This file is auto generated, we're not supposed to modify this file.